app/src/main/java/br/ufg/inf/pes/healthhelp/dao/UsuarioDAO.java

```diff
@@ -41,7 +47,33 @@ public void inativar(Usuario usuario) {
         //TODO
     }
 
-    public void buscarPorLogin(String login, String senha) {
+    public void buscarPorLogin(final String login, final String senha) {
         //TODO
+        //Stub implementation
+        new Thread(new Runnable() {
```

Comment:
Isso também não é a melhor forma de se fazer isso. Vai acabar com a memória do celular fácil fácil também. Maneiras melhores de lidar com isso:
[Android Service](https://developer.android.com/reference/android/app/Service.html)
[AsyncTask](https://developer.android.com/reference/android/os/AsyncTask.html) <- Essa foi o que usamos no outro projeto, certeza de funcionar pro que precisamos
[Tutorial com vários outros](http://www.vogella.com/tutorials/AndroidBackgroundProcessing/article.html#usage-of-serviceshttp://www.vogella.com/tutorials/AndroidBackgroundProcessing/article.html#usage-of-services)